firedrake/slate/preconditioners.py

```diff
@@ -217,6 +217,11 @@ def initialize(self, pc):
         hdiv_projection_ksp.setFromOptions()
         self.hdiv_projection_ksp = hdiv_projection_ksp
 
+        # Switch to using the reconstructor over a KSP to get the
+        # HDiv conforming solution
+        opts = PETSc.Options()
+        self.recon_flag = bool(opts.getBool(prefix + "use_reconstructor", False))
+
```

Comment:
For some reason, `getBool` returns `1` instead of `True` when switched on. This is why I cast to `bool` here.